config/networking/remove-resource-requirements.yml

```diff
@@ -0,0 +1,25 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+#@ if data.values.remove_resource_requirements:
```

Comment:
maybe add an `or data.values.patch_for_kind` ?

Reply:
I think we came to the same conclusion as a team.